source/js/donate-modal/donate-modal.scss

```diff
@@ -97,5 +107,13 @@
     font-weight: 400;
 
     @include scaleText(22px, 28px, 22px, 28px);
+
+    @media (max-width: $bp-sm) {
+      font-family: "Nunito Sans", Helvetica, Arial, sans-serif;
+      font-size: 16px;
+      line-height: 1.5;
```

Comment:
we don't need
```css
font-size: 16px;
line-height: 1.5;
```
having the following line should be sufficient
```css
 @include scaleText(16px, 24px);
```

Reply:
oh nice!